cmd/src/cli/run.rs

```diff
@@ -65,11 +65,30 @@ pub fn run(package: bool, port: u16, persist: bool) -> DefaultResult<()> {
         rules,
     };
 
+    let n3h_path = env::var("HC_N3H_PATH").ok();
+    let n3h_mode = env::var("HC_N3H_MODE").ok();
+    let n3h_persistence_path = env::var("HC_N3H_WORK_DIR").ok();
+
+    // network config
+    let network_config = if networked || n3h_path.is_some() {
```

Comment:
Why is it relevant, wether the n3h path is set when we will take the default anyway?